build.sbt

```diff
@@ -107,6 +107,19 @@ lazy val catsInterop = crossProject(JSPlatform, JVMPlatform)
 lazy val catsInteropJVM = catsInterop.jvm
 lazy val catsInteropJS  = catsInterop.js
 
+lazy val monixInterop = project
+  .in(file("interop/monix"))
+  .settings(name := "caliban-monix")
+  .settings(commonSettings)
+  .settings(
+    libraryDependencies ++= Seq(
+      "dev.zio"  %% "zio-interop-reactivestreams" % "1.0.3.5-RC3",
+      "dev.zio"  %% "zio-interop-cats"            % "2.0.0.0-RC10",
```

Comment:
`"dev.zio"  %% "zio-interop-monix"           % "3.1.0.0-RC1",`

Reply:
I initially did but the code just uses a Future as the intermediate type, so I’m not sure it’s faster and you lose the interruptibility, don’t you?

Reply:
Ah I didn't know it was like that. :+1: